includes/content-distribution/class-outgoing-post.php

```diff
@@ -154,6 +150,10 @@ public static function validate_distribution( $urls ) {
 	 * @return array|WP_Error Config array on success, WP_Error on failure.
 	 */
 	public function set_distribution( $site_urls ) {
+		if ( empty( $site_urls ) ) {
+			return new WP_Error( 'no_site_urls', __( 'No site URLs provided.', 'newspack-network' ) );
+		}
+
```

Comment:
The empty URL validation moved from `validate_distribution()` to here because removing a distribution can result in an empty array and should be valid.